src/python_framework_v02/troute/nhd_network.py

```diff
@@ -42,7 +42,7 @@ def out_degrees(N):
     return in_degrees(reverse_network(N))
 
 
-def extract_connections(rows, target_col, terminal_code=0):
+def extract_connections(rows, target_col, terminal_codes={0}):
```

Comment:
Something like below would be a more defensive way and not require the user to worry to about what type of collection is passed. If the user doesn't pass an iterable, an error will be raised when the set is created.

```python
def extract_connections(rows, target_col, terminal_codes=None):
    if terminal_codes is not None and isinstance(terminal_codes, Iterable):
        terminal_codes = set(terminal_codes)
    else:
        terminal_codes = set()
```